tower.yml

```diff
@@ -0,0 +1,5 @@
+reports:
+  multiqc_report.html:
+    display: "MultiQC HTML report"
+  samplesheet.csv:
+    display: "Auto-created samplesheet with collated metadata and FASTQ paths"
```

Comment:
We could add the paths of other output files which are interesting to visualize on Tower

Reply:
Pin [this](https://github.com/nf-core/crisprseq/pull/130/files) PR for reference

Reply:
I have checked and I don't see where we could use this feature.